text/0002-split-coinstake-transaction/0002-split-coinstake-transaction.md

```diff
@@ -8,25 +8,23 @@
 - Author: hrobeers
 
 ## Summary
-Splitting the coinstake transaction into a *monetary creation* and a *coin-age consumption* transaction allows multiple improvements on peercoin's protocol.
-Apart from aligning the coin creation with Proof-of-Work blocks, it does enable to following changes:
+Splitting the coinstake transaction into a *monetary creation* and a *coin-age consumption* transaction tunes the Peercoin protocol to more closely align with Bitcoin's format.
+By putting the block reward back into the coinbase transaction, the coinstake transaction can be signed in advance while the coinbase transaction is signed at the time of block creation.
```

Comment:
I think this should be in the Motivation section, or at least mentioned there since it's one of the motivations.

And it should probably be noted that it's a minor benefit. The scope of affected tools is very small (much smaller than the timestamp removal).